diff --git a/js/src/bybit.js b/js/src/bybit.js
index 57d5c70479c833c8a3ca74b36f2809895eccd14b..3c2c169525eff42b6c48a04b281dffbea1a16205 100644
--- a/js/src/bybit.js
+++ b/js/src/bybit.js
@@ -2169,6 +2169,7 @@ export default class bybit extends Exchange {
         const result = this.safeDict(response, 'result', {});
         const tickers = this.safeList(result, 'list', []);
         const rawTicker = this.safeDict(tickers, 0);
+        rawTicker.time = response.time;
         return this.parseTicker(rawTicker, market);
     }
     async fetchTickers(symbols = undefined, params = {}) {
@@ -2273,6 +2274,9 @@ export default class bybit extends Exchange {
         //         "time": 1672376496682
         //     }
         //
+        for (const r of Object.values(response.result.list)) {
+          r.time = response.time;
+        }
         const result = this.safeDict(response, 'result', {});
         const tickerList = this.safeList(result, 'list', []);
         return this.parseTickers(tickerList, parsedSymbols);
